x/ecocredit/core/msg_bridge_test.go

```diff
@@ -153,34 +158,17 @@ func TestMsgBridge(t *testing.T) {
 				},
 				Target:    "polygon",
 				Recipient: addr1,
-				Contract:  contract,
-			},
-			expErr: true,
-		},
-		"invalid bridge target": {
-			src: MsgBridge{
-				Owner: addr1,
-				Credits: []*Credits{
-					{
-						BatchDenom: batchDenom,
-						Amount:     "10",
-					},
-				},
-				Target:    "polygon1",
-				Recipient: recipient,
-				Contract:  contract,
 			},
-			expErr: true,
+			expErr:    true,
+			expErrMsg: "not a valid ethereum address",
 		},
 	}
 
-	for msg, test := range tests {
-		t.Run(msg, func(t *testing.T) {
-			t.Parallel()
```

Comment:
Heads up. This was causing the same test to be run multiple times and not the others.

Reply:
Also see https://github.com/regen-network/regen-ledger/pull/1230#discussion_r912325613

Reply:
Reverted in https://github.com/regen-network/regen-ledger/pull/1226/commits/70e569a47aacc96a724faca95d51bf71a3c87c58. This caused several errors unrelated to the scope of this pull request. Opening a separate pull request to address and will likely migrate to gherkin/gocuke.

Reply:
Resolved in #1231